les/serverpool.go

```diff
@@ -76,16 +76,24 @@ var (
 	sfSelected      = utils.NewNodeStateFlag("selected", false, false)
 	sfDialed        = utils.NewNodeStateFlag("dialed", false, false)
 	sfConnected     = utils.NewNodeStateFlag("connected", false, false)
-	sfRedialWait    = utils.NewNodeStateFlag("redialWait", false, true)
```

Comment:
I don't think it's necessary to save "Redial-Wait" status after shutdown. It's totally fine to re-dial after restart.